docs/api-development/Asynchronous APIs/01-Introduction.md

```diff
@@ -26,7 +26,7 @@ Asynchronous APIs are appropriate to use when designing an API to meet any of th
 - When performing a request that may take a long period of time, such as large data exports, human task interrupted workflows or analytic workloads
 - When timeliness of state changes are critical for the use case, such as near-real-time updates from IoT devices
 - When designing a decoupled system, such as in a microservices environment
-- When there is a requirement to use/display the most up to date copy of data
+- When it is important for the API Consumer to use the most up to date version of a resource, as they can be notified of a change, as opposed to querying the source system each time it is used
```

Comment:
```suggestion
- When it is important for the API Consumer to use the most up to date version of a resource, as they can be notified of a change rather than querying the source system each time it is used
```